src/AutoRest.CSharp/Mgmt/Generation/ResourceContainerWriter.cs

```diff
@@ -155,29 +160,26 @@ private void WriteTryGetVariants(RestClientMethod getMethod)
 
             _writer.Line();
             _writer.WriteXmlDocumentationSummary($"Tries to get details for this resource from the service.");
-            WriteContainerMethodScope(false, $"{_resource.Type.Name}", "TryGet", passThruParameters, writer =>
+            WriteContainerMethodScope(false, $"Response<{_resource.Type.Name}>", "GetIfExists", passThruParameters, writer =>
```

Comment:
Actually we have an extension method to wrap something with Response
```suggestion
            WriteContainerMethodScope(false, $"{_resource.Type.WrapResponse()}>", "GetIfExists", passThruParameters, writer =>
```

Reply:
Personally I think `$"Response<{_resource.Type.Name}>"` has better readability.